packages/base/src/commands.ts

```diff
@@ -410,6 +410,35 @@ export function addCommands(
     },
   });
 
+  commands.addCommand(CommandIDs.boundingBoxes, {
+    label: trans.__('Add Bounding Boxes'),
```

Comment:
```suggestion
    label: trans.__('Generate Bounding Boxes'),
```

Sorry for the minor nitpick :)

Reply:
We should probably homogenize this, we have currently "Extract Centroids", so should it be "Extract Bounding Boxes" ?

Reply:
Otherwise, we can also follow exactly the processing naming from QGIS so that users know what to expect

Reply:
Qgis has `Centroids` & `Bounding Boxes`. Yes, We can keep the same.
![59235](https://github.com/user-attachments/assets/985455c3-92ac-4c4e-aea3-440bac441b9d)